bin/configs/csharp-httpclient-net9.yaml

```diff
@@ -1,8 +1,8 @@
 # for .net standard httpclient
-generatorName: csharp
+generatorName: csharp-restsharp
```

Comment:
Should the config name be updated to `csharp-restsharp-net9` when it actually is using the `csharp-restsharp` generator? Or maybe this change wasn't intentional and it should be kept to `csharp`?